ansible/roles/master/tasks/localBuildInstall.yml

```diff
@@ -9,6 +9,7 @@
     - kube-scheduler
     - kube-controller-manager
     - kubectl
+    - kubelet
```

Comment:
You'll have to explain why the master must be most of a node (or maybe I'll get it when I read more)